pegasus/sites.v3/code.org/views/minecraft_2018.haml

```diff
@@ -28,7 +28,7 @@
         .mc-tutorial-info.mc-tutorial-info-wide
           .mc-tutorial-details
             %h2.mc-h1-second-box
-              = hoc_s(:hoc2018_minecraft_aquatic_title1)
+              = I18n.t(:hoc2018_mc_aquatic_title)
               %br/
               = hoc_s(:hoc2018_minecraft_aquatic_title2)
```

Comment:
can this be removed?

Reply:
Good question!  The translation for "Minecraft voyage aquatic" is associated with the key "hoc2018_mc_aquatic_title" in Crowdin.  "hoc2018_minecraft_aquatic_title1" key only existed in the haml file.

Reply:
I'm specifically referring to `hoc2018_minecraft_aquatic_title2` here

Reply:
Yes, it can be removed as well.